superset/templates/appbuilder/navbar_right.html

```diff
@@ -24,15 +24,26 @@
     {% set locale = 'en' %}
 {% endif %}
 
-{% if not current_user.is_anonymous %}
+{% set can_sqlquery = appbuilder.sm.can_access('can_sqllab', 'Superset') %}
```

Comment:
Looks like you've got changes from another PR in here, maybe checkout master then rebase your feature here on top of it?